src/actions/trigger-action/index.ts

```diff
@@ -14,6 +14,7 @@ interface Context {
   payload: {
     inputs: {
       email: string
+      repository: string
       event: string
       param: string
```

Comment:
`triggerAction` is a generic endpoint for multiple operations, and I feel like we are adding an extra `repository` param for only a single operation without regard for the others. This is possibly fine to do once, but I feel it gets a bit messy when we have to add another param for a different operation, and so on - soon we have a `payload` that tries to cater to every different possible operation.

A cleaner approach might perhaps be to pass a JSON string in the `param` with all the info you need (`issueKey` and `repository`). Alternatively, we might be able to identify what repository the action is running in from the environment somehow, and not need to pass it at all.

That contradicts the comment I made [in the slack-bot PR](https://github.com/Shuttlerock/slack-bot/pull/142#discussion_r612962380), but it is worth investigating if an action is aware of its repository I think? It would simplify this a lot.